test-it/ViaTests/ScanFormatErr.via

```diff
@@ -0,0 +1,220 @@
+// Test scan from string
```

Comment:
We should create VI tests in Desktop for different error cases (with a subVI call) and validate those with this PR to make sure they all pass.